### DISCUSSION TITLE
Where to place extra upscale models?

### DISCUSSION THREAD

**Discussion by k8kiss** [lllyasviel/stable-diffusion-webui-forge#183] [Q&A] [answered]
Hello everyone!
Where to place extra upscale models? I don't see an "upscale models" in the models folder like on A1111. I can see upscale models but I have no idea where to place new ones!
Thanks in advance for nay help!

**Comment by ArnorWing** [comment]
Just make a folder for them

**Reply by k8kiss** [reply]
Thanks, but still confused. In models folder? How do I map it? or just call it like on A1111 (ESRGAN)?

**Reply by MNeMoNiCuZ** [reply, marked as answer]
Yes. ESRGAN or GFPGAN works fine.
ESRGAN for most models if you don't know what models you have.

**Reply by k8kiss** [reply]
Thank you.. I will try soon.
On another topic... can you please take a look at this thread in case you have been able to use Photomaker
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/65
Maybe you can help us figure out why Photomaker is not working... many thanks in advance!

**Reply by MNeMoNiCuZ** [reply]
This is too technical for me, I likely won't be able to help there.
I'll take a peek though and see.

**Reply by k8kiss** [reply]
Really appreciate your help!! Creating a ESRGAN folder in Models worked and my extra models are now available!
